simplefs/simplefs.go

```diff
@@ -476,12 +476,29 @@ func (k *SimpleFS) updateWriteProgress(
 	}
 }
 
+var filesToIgnore = map[string]bool{
+	".Trashes":   true,
+	".fseventsd": true,
+	".DS_Store":  true,
+}
+var prefixesToIgnore = []string{"._"}
```

Comment:
On the one hand, for performance ideally this would be a regex instead of a loop. On the other, regexes are less easily debugged, and this set of prefixes will likely always remain small.

Reply:
Hmm I guess if it grows large we could make it a trie? It feels to me that regex is gonna be slower to do a simple prefix match. Not sure!

Reply:
Naw, regexes compile down to a state machine that would be functionally equivalent to a trie for cases where a trie would help.